packages/salesforcedx-vscode-apex/README.md

```diff
@@ -35,6 +37,13 @@ The Problems view in the bottom pane also lists the syntax errors. Double-click
 The Apex outline view shows the structure of the Apex class or trigger that’s open in the editor. For a list of the symbols in your file, press Cmd+Shift+O (macOS) or Ctrl+Shift+O (Windows or Linux). To jump to one of the symbols, select it in the list.  
 ![Outline view, showing the symbols in an Apex class](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/images/apex_outline.png)
 
+## Enable Code Smartness for SObjects
+To activate this extension’s code smartness features for standard and custom objects and their fields, including for custom fields on standard objects, press Cmd+Shift+P (macOS) or Ctrl+Shift+P (Windows or Linux) and then select **SFDX: Refresh SObject Definitions** from the command palette. 
```

Comment:
Because you have quite a few conjunctions here, I suggest adding a comma to clarify what goes with what. "objects, press Cmd+blah (macOS) or Ctrl+blah (Windows or Linux), and then select blah..."